src/geocoder.js

```
@@ -28,14 +37,18 @@ export default class Geocoder {
     }
 
     this.timeBetweenRequests =
-      options.clientId && options.privateKey ? 100 : 200;
+      options.clientId && options.privateKey || options.apiKey ? 100 : 200;
```

So, actually it should throw an error, when no key or client ID was passed.